main.py

```diff
@@ -250,6 +250,12 @@ def infer_worker(gpu, ngpu_per_node, args, res_queue=None):
     infer_dataset = DatasetInfer(args.infer_files)
     infer_sampler = None
 
+    # Check that the intervals file and infer data have same number of entries.
+    intervals = pd.read_csv(args.intervals_file, sep='\t', header=None, names=['chrom', 'start', 'end'],
```

Comment:
Why do we need to distribute this check to every GPU? Would it be better placed in `main()`?

Reply:
Good point. Moved it over

Reply:
done